### PR TITLE
[15.0][IMP] hr_timesheet_sheet_period: add Current Period filter in timesheets

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
When being a manager and seeing All timesheets for all employees it is useful to filter the ones belonging to the current period:

image

cc @ForgeFlow

### CODE REVIEW

**Review by GuillemCForgeFlow — APPROVED** [review]
LGTM 👍🏿

**Review by JordiMForgeFlow — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]

File: hr_timesheet_sheet_period/views/hr_timesheet_sheet_view.xml
Comment (alexey-pelykh) [inline hr_timesheet_sheet_period/views/hr_timesheet_sheet_view.xml]: What's the point in renaming?

**Review by alexey-pelykh — APPROVED** [review]
LGTM with one question

File: hr_timesheet_sheet_period/views/hr_timesheet_sheet_view.xml
Comment (AaronHForgeFlow) [reply hr_timesheet_sheet_period/views/hr_timesheet_sheet_view.xml]: When there are issues with a view it is easier to identify the view when the name has a clue in it. I can leave it as it was if you think this is not appropriate.

Comment (alexey-pelykh) [reply hr_timesheet_sheet_period/views/hr_timesheet_sheet_view.xml]: I don't think it's appropriate since the ids are scoped to the modules anyways, yet it's a non-blocking opinion. I'll leave it to other maintainers :)

Comment (AaronHForgeFlow) [reply hr_timesheet_sheet_period/views/hr_timesheet_sheet_view.xml]: For me it is not a must thing, I just left it to what it was

**Review by alexey-pelykh — APPROVED** [review]
LGTM

**Pull request merged**
